<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>General settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <property name="rowWrapPolicy">
           <enum>QFormLayout::DontWrapRows</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;New admin password:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>adminPasswordEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="adminPasswordEdit">
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&amp;Repeat new admin password:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>adminPasswordRepeatEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="adminPasswordRepeatEdit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Max &amp;number of songs in playlist:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>maxPlaylistSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QSpinBox" name="maxPlaylistSpinBox">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Up</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Down</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>&amp;Exclude long tracks from collection:</string>
            </property>
            <property name="buddy">
             <cstring>excludeLongTracksCheckbox</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QSpinBox" name="maxTrackLengthSpinBox">
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::NoButtons</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="maxTrackLengthUp">
              <property name="text">
               <string>Up</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="maxTrackLengthDown">
              <property name="text">
               <string>Down</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="excludeLongTracksCheckbox">
            <property name="text">
             <string>(Restart required)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="maxTrackLengthLabel">
            <property name="text">
             <string>Maximum track &amp;length in minutes:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>maxTrackLengthSpinBox</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>&amp;Start/Quit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Behaviour on start &amp;&amp; quit</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="1" column="1">
           <widget class="QCheckBox" name="fullscreenOnStartCheckBox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>&amp;Autostart MPD playback when connected:</string>
            </property>
            <property name="buddy">
             <cstring>playOnConnectCheckBox</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="playOnConnectCheckBox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>St&amp;op MPD playback on quit:</string>
            </property>
            <property name="buddy">
             <cstring>stopOnQuitCheckBox</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="stopOnQuitCheckBox">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Start in &amp;fullscreen:</string>
            </property>
            <property name="buddy">
             <cstring>fullscreenOnStartCheckBox</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>&amp;Virtual keyboard</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Virtual keyboard keys (restart required)</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Keys on row 3:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="vkRow3Edit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="vkRow2Edit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Keys on row 2:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="vkRow1Edit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Keys on row 1:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="vkRow4Edit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Keys on row 4:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>&amp;MPD Server</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>MPD server configuration (restart required)</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>MPD &amp;Server:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>mpdServerEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="mpdServerEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>MPD P&amp;assword:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>mpdPasswordEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="mpdPasswordEdit">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>MPD &amp;Port:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>mpdPortEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="mpdPortEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>okBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>cancelBtn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>maxPlaylistSpinBox</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>maxPlaylistSpinBox</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>490</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxTrackLengthUp</sender>
   <signal>clicked()</signal>
   <receiver>maxTrackLengthSpinBox</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxTrackLengthDown</sender>
   <signal>clicked()</signal>
   <receiver>maxTrackLengthSpinBox</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excludeLongTracksCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxTrackLengthLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excludeLongTracksCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxTrackLengthSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excludeLongTracksCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxTrackLengthUp</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excludeLongTracksCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxTrackLengthDown</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(QWidget*)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>tabChanged(QWidget*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>okBtn()</slot>
  <slot>cancelBtn()</slot>
  <slot>tabChanged(QWidget*)</slot>
 </slots>
</ui>
